tiatoolbox/utils/misc.py

```diff
@@ -567,6 +576,29 @@ def unzip_data(zip_path, save_path, del_zip=True):
         os.remove(zip_path)
 
 
+def save_dict_to_json(input_dict, save_path):
```

Comment:
Renamed to `save_as_json` and allow list as input. However I'm worried this will create misusage later on because this is very primitive because it only converts np.ndarray, other type is passed through